server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodServer.java

```diff
@@ -320,10 +326,12 @@ private void addSelfToTopologyView(EmbeddedCacheManager cacheManager) {
       topologyChangeListener = new ReAddMyAddressListener(addressCache, clusterAddress, address);
       addressCache.addListener(topologyChangeListener);
 
+      timeService = cacheManager.getGlobalComponentRegistry().getTimeService();
```

Comment:
I'm surprised `SecurityActions.getGlobalComponentRegistry()` isn't require here :)

Reply:
Probably is, but we don't have that great of test coverage for auth :D